.devcontainer/linux-win/devcontainer.json

```
@@ -18,5 +18,7 @@
   },
   "remoteUser": "simbricks",
   "onCreateCommand": "sudo make -C /simbricks convert-images-raw",
-  "postStartCommand": "sudo chmod o+rw /dev/kvm "
+  "postStartCommand": "sudo chmod o+rw /dev/kvm ",
+  "workspaceMount": "source=${localWorkspaceFolder},target=/workspaces/simbricks-examples,type=bind",
```

```suggestion
  "workspaceMount": "source=${localWorkspaceFolder},target=/w/examples,type=bind",
```
I suggest we go with the same as for coder to reduce path length as much as possible to not run into issues with sockets.